src/dxvk/imgui/dxvk_imgui.cpp

```diff
@@ -1667,6 +1669,16 @@ namespace dxvk {
           }
         }
       }
+
+      if (legacyTextureGuiShowAssignedOnly()) {
+        if (std::string(uniqueId) != "textures") {
```

Comment:
Changing `std::string` to `strcmp(..) == 0` (or to `std::string_view`) is preferable to not create `std::string` instance.

(Optionally, might need to put `assert(..)` + `return` to ensure that `uniqueId` is not null at the beginning of the function, so it's more obvious when debugging)

Reply:
I didn't know about `std::string_view`. Really useful thank you!